src/coprocessor/statistics/analyze.rs

```diff
@@ -359,7 +359,7 @@ impl<S: Snapshot> RowSampleBuilder<S> {
             columns_info,
             column_groups: req.take_column_groups().into(),
             quota_limiter,
-            is_quota_auto_tune,
+            is_auto_analyze: is_quota_auto_tune,
```

Comment:
Could you please explain a bit more about the name? To my understanding, auto analyze is the source type of TiDB analyze. How is it related to the quota auto tune?

I can understand the old name, but the new name is a bit confusing to me.

Reply:
fixed the typo.